src/backend/marsha/core/models/site.py

```diff
@@ -101,6 +101,20 @@ class SiteConfig(BaseModel):
         blank=True,
     )
 
+    homepage_banner_title = models.CharField(
+        max_length=255,
+        verbose_name=_("Homepage banner title"),
+        help_text=_("Homepage banner title"),
+        blank=True,
+        null=True,
+    )
+    homepage_banner_text = models.TextField(
+        verbose_name=_("Homepage banner text"),
+        help_text=_("Homepage banner text"),
+        blank=True,
+        null=True,
+    )
+
```

Comment:
I think we should move those attribute under `logo_url`  to get a more consistent admin view:
![image](https://github.com/openfun/marsha/assets/720491/c1e224b5-7f45-4b05-8d14-7d09cb9a011d)
WDYT ?

Reply:
Also, we can add some generated values here https://github.com/openfun/marsha/blob/master/src/backend/marsha/development/management/commands/load_development_datasets.py#L151-L152

Reply:
Done